mailit/tests/email_parser/incoming_mail_tests.py

```diff
@@ -368,6 +368,36 @@ def test_it_marks_the_outbound_message_as_an_error(self):
         outbound_message = OutboundMessage.objects.get(id=self.outbound_message.id)
         self.assertEquals(outbound_message.status, "error")
 
+class FlaggingBouncedMail(TestCase):
+    def setUp(self):
+        super(FlaggingBouncedMail, self).setUp()
+        self.message = Message.objects.get(id=1)
+        self.contact = Contact.objects.get(value="mailnoexistente@ciudadanointeligente.org")
```

Comment:
Minor but what's with the `mailnoexistente@ciudadanointeligente.org` address? Can it be something .test to guarantee no email goes anywhere?

Reply:
This is the pombola default tests. We didn't write these. So I think best left as is for now.